packages/dashboard/src/lib/components/shared/configurable-operation-arg-input.tsx

```
@@ -10,42 +19,374 @@
+                return (
+                    <CurrencyInput
+                        definition={definition}
+                        value={value}
+                        onChange={onChange}
+                        readOnly={readOnly}
+                    />
+                );
+
+            default:
+                // Try to use the component registry for other UI components
+                const componentId = UI_COMPONENT_MAP[uiComponent as keyof typeof UI_COMPONENT_MAP];
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)